productcomments.php

```diff
@@ -969,6 +968,23 @@ private function renderProductCommentsList($product)
         $commentsNb = $productCommentRepository->getCommentsNumber($product->id, (bool) Configuration::get('PRODUCT_COMMENTS_MODERATE'));
         $isPostAllowed = $productCommentRepository->isPostAllowed($product->id, (int) $this->context->cookie->id_customer, (int) $this->context->cookie->id_guest);
 
+        /* generate pagination */
+        $commentsNav = '';
+        $commentsTotalPages = 0;
+        $commentsPerPage = (int) Configuration::get('PRODUCT_COMMENTS_COMMENTS_PER_PAGE');
+        if ($commentsNb > 0) {
+            $commentsTotalPages = ceil($commentsNb / $commentsPerPage);
+            $commentsNav .= '<ul>';
+            $prevCount = 0;
+            $commentsNav .= '<li data-page="' . $prevCount . '" id="pcl_page_' . $prevCount . '"><span class="prev"><i class="material-icons">chevron_left</i></span></li>';
```

Comment:
maybe it would be better to have a dedicated template for `$commentsNav` rather than writing serious HTML in PHP?

Reply:
Done. This pure HTML is moved to TPL.

Reply:
thank you